en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
+#### API management
 
-#### Scenario 
- 
-Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
+Choreo supports the following API management features for all the plans:
 
-The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights<br/>(with only 7 days of data retention) | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights |
 
-Having the jet fuel price, the scheduler is now able to calculate the airfare for each route. The Scheduler retrieves the airline routes from the [Airline Routes API](#airlineRoutesAPI). Upon receiving the airline routes, it iterates over each route, computes the airfare, and updates the AirFare database.
 
-#### Glossary 
+#### Monitoring and observability
 
-- <a class="pricing" id="scheduler">Scheduler</a>: A scheduled task created on Choreo that runs every day.
-- <a class="pricing" id="oilPriceAPI">Oil Price API</a>: An external-facing API that returns the crude oil price for the day.
-- <a class="pricing" id="airFareDatabase">Air Fare Database</a>: The database that stores the jet fuel price which is used to compute the airfare for each route.
-- <a class="pricing" id="airlineRoutesAPI">Airline Routes API</a>: An external-facing API that provides details about the airline routes and a list of airline routes.
-- <a class="pricing" id="airlineRoutesDatabase">Airline Routes Database</a>: The database that stores the airline routes information.
+Choreo supports monitoring and observability features for each plan as follows:
 
-<br/>
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| In-built monitoring dashboard<br/><br/>Logs<br/>(7 days of data retention)<br/><br/>Traces<br/>(7 days of data retention)<br/><br/>Alerting<br/>(7 days of data retention) | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties |
```

Comment:
The rendered table is a little off

Reply:
Other plans how many days of data retention?

Reply:
> The rendered table is a little off

It looks as follows in the local build:

<img width="750" alt="Screen Shot 2023-01-05 at 12 00 12" src="https://user-images.githubusercontent.com/26239861/210716229-9bd8d868-57c9-46e7-96d1-22c3b552a283.png">

Reply:
> Other plans how many days of data retention?

Updated this based on the latest information received from Rukshan and TharinduW

Reply:
> Other plans how many days of data retention?

Updated